vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -87,13 +108,28 @@
                     $(".open").css('display', 'none');
                     $(".open").css('overflow-y', 'scroll');
                     $(".open").css('height', '400px');
+                } else if(divName.includes("SpaceDivStaff")){
+                    let spaceId = e.target.closest('div').id;
+                    $(e.target).closest('.SpaceDivStaff').addClass("open");
+                    var description = $("div.open p:eq(0)").text();
+                    defaultValue = description;
```

Comment:
what's the point of this?

Reply:
When cancel button is clicked from edit space block or text block, the boxes are re-updated with the title that was before editing it, to save making a call to the backend. 
This is there even in line 102 which was from a previous ticket and I have used the same logic for this ticket as well